app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -383,14 +386,28 @@ class AttendeeFragment : Fragment() {
             } else {
                 attendees.addAll(attendeeRecyclerAdapter.attendeeList)
             }
-            val country = if (country.text.isEmpty()) country.text.toString() else null
-            attendeeViewModel.createAttendees(attendees, country, paymentOptions[selectedPaymentOption])
 
-            attendeeViewModel.isAttendeeCreated.observe(this, Observer { isAttendeeCreated ->
-                if (isAttendeeCreated && selectedPaymentOption == paymentOptions.indexOf(getString(R.string.stripe))) {
-                    sendToken()
+            for (i in 0 until attendees.size) {
```

Comment:
Why do you need i?

Also, all logic should be in ViewModel

Reply:
Okay will close this request and post another one with logic in ViewModel